controller/src/test/java/io/pravega/controller/task/Stream/StreamMetadataTasksTest.java

```diff
@@ -960,7 +960,8 @@ public void sealStreamWithTxnTest() throws Exception {
                 .get().getKey();
```

Comment:
I'm not seeing `getCurrentSegmentSizes` declared in this PR being unit tested.

Reply:
Its merely calling getSegmentOffset which is tested.